src/graph/InsertEdgeExecutor.h

```diff
@@ -30,7 +30,7 @@ class InsertEdgeExecutor final : public Executor {
     StatusOr<std::vector<storage::cpp2::Edge>> prepareEdges();
 
 private:
-    using EdgeSchema = std::shared_ptr<const meta::SchemaProviderIf>;
+    using EdgeSchema = std::shared_ptr<const meta::NebulaSchemaProvider>;
```

Comment:
why replace `SchemaProviderIf` with `NebulaSchemaProvider` ?

Reply:
because the return gets from the cache is `NebulaSchemaProvider`, and `getDefaultValue`  is the function of `NebulaSchemaProvider`.